src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -563,11 +589,58 @@ public void ProcessCommand(IExecutionContext context, string inputLine, ActionCo
             context.AddIssue(issue);
         }
 
+        public static void ValidateLinesAndColumns(ActionCommand command) 
+        {
+            command.Properties.TryGetValue(IssueCommandProperties.Line, out string line);
+            command.Properties.TryGetValue(IssueCommandProperties.EndLine, out string endLine);
+            command.Properties.TryGetValue(IssueCommandProperties.Column, out string column);
+            command.Properties.TryGetValue(IssueCommandProperties.EndColumn, out string endColumn);
+
+            var hasStartLine = int.TryParse(line, out int lineNumber);
+            var hasEndLine = int.TryParse(endLine, out int endLineNumber);
+            var hasStartColumn = int.TryParse(column, out int columnNumber);
+            var hasEndColumn = int.TryParse(endColumn, out int endColumnNumber);
+            var hasColumn = hasStartColumn || hasEndColumn;
+
+            if (hasEndLine && !hasStartLine)
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'end_line' can only be set if 'line' is provided");
+            }
+
+            if (hasEndColumn && !hasStartColumn)
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'end_column' can only be set if 'col' is provided");
+            }
+
+            if (!hasStartLine && hasColumn) 
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'column' and 'end_column' can only be set if 'line' value is provided."); 
+            }
+
+            if (hasEndLine && line != endLine && hasColumn) 
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'column' and 'end_column' cannot be set if 'line' and 'end line' are different values."); 
+            }
+
+            if (hasStartLine && hasEndLine && endLineNumber < lineNumber) 
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'end_line' cannot be less than 'line'."); 
+            }
+
+            if (hasStartColumn && hasEndColumn && endColumnNumber < columnNumber) 
+            {
+                throw new Exception($"Invalid {command.Command} command value. 'end_column' cannot be less than 'col'."); 
+            }    
+        }
+
         private static class IssueCommandProperties
         {
             public const String File = "file";
             public const String Line = "line";
+            public const String EndLine = "end_line";
```

Comment:
do we have precedent for compound words?

Reply:
Wonder whether we should use `-` or camel case. For example, consider:

`::set-foo my_prop=val, my_prop2=val2:some data`

idk the right answer. Personally think camel case maybe? (2 cents)

If camel case, make sure the comparison is ordinalignorecase